WootricSDK/WootricSDK/Wootric.h

```diff
@@ -47,6 +47,12 @@
  @param viewController View controller in which you would like to display the survey.
 */
 + (void)showSurveyInViewController:(UIViewController *)viewController;
+/**
+ @discussion It shows survey if end user is eligible.
+ @param viewController View controller in which you would like to display the survey.
+ @param eventName Event Name.
+*/
++ (void)showSurveyInViewController:(UIViewController *)viewController event:(NSString *)eventName;
```

Comment:
Objective's C optional params?

Reply:
Yes, this way we can keep the current existing method.